README.md

```diff
@@ -21,7 +21,7 @@ and then open index.html in your browser.
 👤 **Davis Nyale**
 
 - GitHub: [@DavyKn](https://github.com/DavyKn)
-- Github livesite: ( https://davykn.github.io/my_portfolio/)
+- Github livesite: ( https://davykn.github.io/Davis_Portfolio/)
```

Comment:
- [ **OPTIONAL** ] Good job with your app deployment, now everyone can see your application online.  So, I think it would be better to add the link in the `Live Demo` section as it is his place in the documentation([Microverse Readme template](https://github.com/microverseinc/readme-template/blob/master/README.md)) and I also suggest you use this format below to make your documentation looks smart(as it will not show the link, but the name in the [square brackets]). For an organized Readme template, you can check [here](https://github.com/tresorsawasawa/MyPortfolio/blob/develop/README.md) and improve all sections in your documentation.
```bash
[Live Demo Link](https://davykn.github.io/Davis_Portfolio/)
```